roles/virtualmachine-apb/templates/vm-linux.yml.j2

```diff
@@ -22,7 +22,7 @@ spec:
   volumes:
     - name: registryvolume
       registryDisk:
-        image: kubevirt/fedora-cloud-registry-disk-demo:latest
+        image: kubevirt/{{ vmdiskimage }}-registry-disk-demo:latest
```

Comment:
This template is about to be replaced by a more suitable one based on output of the virt team.

Regarding disks - the idea was to allow more flexibility in image selection. 
With this change, the `Linux` VM would be created just from one of the hard-coded option.

@aglitke, @fabiand, what is expected to be supported?